### DISCUSSION TITLE
Marketing site & Docs site discussion

### DISCUSSION THREAD

**Discussion by daniel-farina** [osmosis-labs/feature-requests#49] [General]
This is a place to discuss the best way to integrate/reference the documentation site on the new marketing site that is being worked on. I created this discussion so that we can all make points, raise questions and find answers. 

The current doc site lives here: 
https://osmosis-labs.github.io/osmosis/

Is there a link for the marketing site? 
What are some of the questions/concerns of the marketing DAO team? 

Thank you!

**Comment by czarcas7ic** [comment]
From my understanding we have something as follows:

An osmosis landing page, whose purpose is to be visually appealing and attract users (run by MDAO?)
An osmosis docs page, whose purpose is to provide technical documentation as well as tutorials
An osmosis chat page, whose purpose is to provide technical live support (run by CSDAO?)

I think having some kind of min viable product from both MDAO and CSDAO (assuming those products are run by those respective DAOs, I'm just going off my memory from prior conversations) would allow us to better understand where potential overlap may be as well as placing wanted features in the proper project. Also would be nice to have a POC from both DAOs for who is specifically tasked with these products so we can sync when needed.

EDIT: I also remember something about us wanted to host the updates from the lab on our own server. So a feature like that would probably be on the landing page side of the house.
